aireview/src/aireview/infrastructure/github_client.py

```diff
@@ -6,12 +6,14 @@
 import logging
 
 from litellm.llms.custom_httpx.http_handler import headers
+from nltk.chunk.named_entity import shape
 from requests import Response
 from typer.cli import state
 
 from aireview.domain.entities.pull_request import PullRequest
```

Comment:
**style**: Duplicate import of 'logging' module. Remove redundant import as it's already imported at line 6